src/app/work-item/work-item.component.html

```diff
@@ -29,8 +29,42 @@ <h2 class="card-pf-title text-center">
     <div class="row toolbar-pf">
       <div class="col-md-12 col-sm-12 col-xs-12">
         <div class="list-action toolbar-pf-actions">
+          <!--Filter-->
+          <div class="form-group toolbar-pf-filter">
+            <label class="sr-only" for="filterwitype">Filter</label>
+            <div class="input-group">
+              <div class="input-group-btn" dropdown>
+                <button class="btn btn-default dropdown-toggle" type="button" id="wi_filter_dropdown" data-toggle="dropdown" dropdown-open>
+                  User
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" role="menu" aria-labelledby="wi_filter_dropdown">
+                  <li *ngIf="!filters.length">
+                    <span>No filters available</span>
+                  </li>
+                  <li *ngFor="let filter of filters;" role="presentation" (click)="activeFilter(filter.id)">
```

Comment:
activeFilter() should not call when the user selects User from the dropdown list.